files/fr/web/html/global_attributes/part/index.html

```diff
@@ -0,0 +1,25 @@
+---
+title: part
+slug: Web/HTML/Global_attributes/part
+browser-compat: html.global_attributes.part
+translation_of: 'Web/HTML/Global_attributes/part'
+---
+<div>{{HTMLSidebar("Global_attributes")}}</div>
+
+<p><a href="/fr/docs/Web/HTML/Global_attributes">L'attribut universel</a> <strong><code>part</code></strong> contient une liste des parties du nom de l'élément, séparés par des espaces. Les éléments du nom permettent de sélectionner et de mettre en forme en CSS des parties spécifiques du <i lang="en">DOM</i> dans l'arbre fantôme avec le pseudo-élément {{CSSxRef("::part")}}.</p>
```

Comment:
typo :) "parties… séparés" > "parties… séparées"

Reply:
When reading the English page, I'd start the second sentence with "Les parties de nom…" instead of "Les éléments du nom…" as "éléments" may be confused wtih HTML elements.

Reply:
LGTM
Fixed in https://github.com/mdn/translated-content/pull/1824/commits/27aba6a7ff86c069ace98c5a4f311eb5753cd272